initLocalTest.sh

```diff
@@ -131,7 +132,7 @@ fi
 set +e
```

Comment:
redundant?

Reply:
Nope, the script will exit if the templates folder is empty (which will be for people running it that don't have access to our configs repo)